aptos-move/aptos-replay-benchmark/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "aptos-replay-benchmark"
+version = "0.1.0"
+description = "A tool to replay and locally benchmark on-chain transactions."
```

Comment:
I think this
```
#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn verify_tool() {
        use clap::CommandFactory;
        Command::command().debug_assert();
    }
}
```
allows us to test commands exhaustively. For CI testing, the plan is to actually use this tool on CI to replay txns and benchmark performance (to track regressions, etc. similar to single-node-performance we have now), so once it is there, it will be "tested" because we use it. Will add a TODO for now, as I will probably add a few options to CLI to download and read transactions/state override.